frontend/src/components/campCreate/CampCreateStep2.vue

```diff
@@ -8,7 +8,8 @@
             v-model="localCamp.campPrototype"
             :vee-rules="{ required: false, excluded: ['', false, true] }"
             :skip-if-empty="false"
-            :name="$tc('entity.camp.prototype')"
+            name="prototype"
+            :label="$tc('entity.camp.prototype')"
```

Comment:
Is it not possible to provide entityName or use an EForm here?

Reply:
Currently this is not an entity field name, and I'm not sure if it should be.